gemfirexd/tools/src/test/java/com/pivotal/gemfirexd/internal/engine/FabricServerTest.java

```diff
@@ -675,27 +675,20 @@ public void testNetworkStartupAPI() throws Exception {
 
     System.setProperty(com.pivotal.gemfirexd.Property.PROPERTIES_FILE, PROP_FILE_NAME);
 
-    if (!ClientSharedUtils.isThriftDefault()) {
-      System.setProperty(Property.START_DRDA, "true");
-    }
-    int port;
+    int port, port2;
     while ((port = AvailablePort.getRandomAvailablePort(AvailablePort.SOCKET))
         <= FabricService.NETSERVER_DEFAULT_PORT);
-    if (!ClientSharedUtils.isThriftDefault()) {
-      System.setProperty(com.pivotal.gemfirexd.Property.DRDA_PROP_PORTNUMBER,
-          String.valueOf(port));
-    }
+    while ((port2 = AvailablePort.getRandomAvailablePort(AvailablePort.SOCKET))
+        <= FabricService.NETSERVER_DEFAULT_PORT);
```

Comment:
@kneeraj addressed the last comment in https://github.com/SnappyDataInc/snappy-store/commit/d88c9d514b8c92def326d2b7790f7e91a2e1d776